scripts/run.sh

```diff
@@ -22,6 +22,7 @@ function cleanup {
 # NATS
 ###
 gnatsd &
+sleep 3
```

Comment:
Try lowering this to `0.1`, `0.5` or `1` - NATS is usually very fast and `0.1` should do. Also add a comment why you are doing this (although seems evident).